owncloudApp/src/main/java/com/owncloud/android/presentation/authentication/oauth/OAuthUtils.kt

```diff
@@ -124,9 +124,10 @@ class OAuthUtils {
             }.build()
 
         fun buildRedirectUri(context: Context): Uri =
-            Uri.Builder()
-                .scheme(context.getString(R.string.oauth2_redirect_uri_scheme))
-                .authority(context.getString(R.string.oauth2_redirect_uri_path))
-                .build()
+            Uri.Builder().apply {
+                scheme(context.getString(R.string.oauth2_redirect_uri_scheme))
+                authority(context.getString(R.string.oauth2_redirect_uri_host))
+                path(context.getString(R.string.oauth2_redirect_uri_path))
+            }.build()
```

Comment:
I think `apply` is not necessary in this case

Reply:
Changed!